Group reflection context docs together and provide new topic on behavior context (#1360)
o3de/o3de.org: content/docs/user-guide/programming/serialization/json-serialize-deserialize.md

---

> title: Serialize and Deserialize JSON Objects
> description: Serialize objects to JSON from the O3DE engine and load them back through deserialization.

Reviewer: ```suggestion
linkTitle: Serialize and Deserialize
title: Serialize and Deserialize JSON Objects
description: Learn how to serialize objects from Open 3D Engine (O3DE) to JSON, and how to load them back through deserialization.
```

weight: 300
---

>  Once a class has been [registered with a serialization context](/docs/user-guide/programming/serialization/register-objects/) objects of that class can be serialized and deserialized. Objects are serialized to JSON with the `AZ::JsonSerialization::Store()` function, and deserialized from JSON with `AZ::JsonSerialization::Load()`.

Reviewer: ```suggestion
After a class is [registered with a serialization context](/docs/user-guide/programming/serialization/register-objects/), you can serialize and deserialize objects of that class. Objects are serialized to JSON with the `AZ::JsonSerialization::Store()` function and deserialized from JSON with `AZ::JsonSerialization::Load()`.
```

Reviewer: In line 9 below this:
* In the first sentence, confirm if "JSON the serializer" is the intended phrasing in the first sentence.
* Change "see" to "refer to".

Author: _"JSON the serializer"_
LOL! Sounds like a wrestler or supervillain.

Reviewer: Line 7/8 edit applied in recent commit. Line 9/10 edits (including our wrestling heel character 😜) still outstanding, in both senses of the word.
